app/javascript/dashboard/components/layout/sidebarComponents/SecondaryChildNavItem.vue

```diff
@@ -112,6 +112,7 @@ $label-badge-size: var(--space-slab);
   padding: var(--space-smaller) var(--space-smaller);
   margin: var(--space-smaller) 0;
   text-align: left;
+  line-height: 1.2;
```

Comment:
Will fix this issue
<img width="191" alt="Screenshot 2022-12-05 at 10 39 33 PM" src="https://user-images.githubusercontent.com/64252451/205699459-6bc8e1b1-1d12-47a6-94a5-735d011b04de.png">